src/UI/TransactionsToastList/components/CustomToast/components/IconToast/components/IconToastFooter/IconToastFooter.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { getIsTransaction } from 'UI/TransactionsToastList/components/CustomToast/helpers';
+import { IconToastPropsType } from '../../iconToast.types';
+import {
+  MessageIconToastFooter,
+  SharedToastFooter,
+  TransactionToastFooter,
+  ComponentToastFooter
+} from './components';
+
+export const IconToastFooter = (props: IconToastPropsType) => {
+  const { transaction, message, component } = props;
+  const isTransaction = transaction && getIsTransaction(transaction);
+
+  if (isTransaction) {
+    return <TransactionToastFooter {...props} />;
+  }
+  if (message) {
+    return <MessageIconToastFooter {...props} />;
+  }
+  if (component !== undefined) {
```

Comment:
component != null

Reply:
this is intentional, a null component means component was defined and has a value, so search for it in the componentDictionary. I will add a comment